.devcontainer/README.md

```diff
@@ -55,3 +55,11 @@ There are other ports needed for the IDE itself but mainly, you only need to wat
 In the docker-compose.yml file:
 - For Python: Change the `VARIANT` argument to the appropirate [variant](https://github.com/microsoft/vscode-dev-containers/tree/main/containers/python-3)
 - For Node: Change the `NODE_VERSION` to the appropriate version used by [nvm](https://github.com/nvm-sh/nvm)
+
+## Using the Devcontainer on macOS
```

Comment:
Could you add this section before the `## Using the Devcontainer` section above.

Reply:
Done